databases/devices_db_schema.sql

```diff
@@ -17,6 +17,7 @@ CREATE TABLE Models (
 	id INT AUTO_INCREMENT PRIMARY KEY
 	, manufacturer INT NOT NULL
 	, category INT NOT NULL
+    , energy_consumption FLOAT NOT NULL
 	, name VARCHAR(200) NOT NULL
 	, comments VARCHAR(1000)
 );
```

Comment:
(cannot post a comment lower in the code)
What do you think about storing the units of the data (jednostki)? e.g. celsius, wats, etc?

Reply:
Well, when more an more parameters are to be introduced then we should change the schema to allow for that (generic database schema).  Do we want to do that? Or do we want to add just a few params to choose from?

Reply:
Add only a few params to choose from ;) I'd rather not make it too complicated